blocks/init/src/blocks/custom/quote/manifest.json

```diff
@@ -12,8 +12,10 @@
     "Quote"
   ],
   "example": {
-    "author": "Narcotics Anonymous",
-    "content": "Insanity is doing the same thing, over and over again, but expecting different results."
+    "attributes": {
+      "author": "Narcotics Anonymous",
```

Comment:
Narcotics Anonymous? 😬 

That's... pretty bleak 👀

Reply:
fixed